javascript/javascript1/week1/HouseyPricey.js

```diff
@@ -0,0 +1,13 @@
+
+function calculateIfhouseIsExpensive (name, wide, deep, hight, gardenSizeInM2, houseCost){
+    const volumeInMeters = wide * deep * hight;
+    const housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeInM2 * 300);
+    if (housePrice > houseCost){
+        console.log(name + " bougth the house for " + houseCost + ". The calculated value of this house is: " + housePrice + ". It was a good purshase!");
+    }
+    else{
+        console.log(name + " bougth the house for " + houseCost + ". The calculated value of this house is: " + housePrice + ". It was a bad purshase!");
+    }
+}
```

Comment:
What if just the right amount was paid? Will the logged message be what you expect?

Reply:
Good question, I haven't considered that, to get the result that I want an equal sign should be added to the comparison in the first line.